frontend/packages/dev-console/integration-tests/support/commands/app.ts

```diff
@@ -1,9 +1,12 @@
 // import { checkErrors } from '../../../../integration-tests-cypress/support';
 
-export {}; // needed in files which don't have an import to trigger ES6 module usage
+export { }; // needed in files which don't have an import to trigger ES6 module usage
 declare global {
   namespace Cypress {
     interface Chainable<Subject> {
+      pageTitleShouldContain(title: string): Chainable<Element>;
+      alertTitleShouldContain(title: string): Chainable<Element>;
+      clickNavLink(path: [string, string]): Chainable<Element>;
```

Comment:
These functions are already available in [integration-tests-cypress](https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/support) folder. So i recommend to re-use them instead of re-creating them

Reply:
while pageTitleShouldContain and alertTitleShouldContain are not present, hence retained. However, I removed implementation for clickNavLink which was present in integration-tests-cypress folder